NestedWrappers.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "Room.h"
+#include "DataPlan.h"
```

Comment:
You aren't using these types in this header file, so just forward declare them.
- [ ] revise.